### DISCUSSION TITLE
'RoIAlignRotated' object has no attribute 'output_size'

### DISCUSSION THREAD

**Discussion by makabakasu** [open-mmlab/mmrotate#167] [Show and tell]
mmcv-1.4.5 is not compatible with mmrotate-0.1.1, resulting in error 'RoIAlignRotated' object has no attribute 'output_size'.
https://github.com/open-mmlab/mmrotate/blob/c5bf348562fd84cc17906c2cf370d1a49fcd3035/docs/en/install.md?plain=1#L17
solutions1： upgrade mmcv
solutions2：
change
```
if isinstance(self.roi_layers[0], ops.RiRoIAlignRotated):
     out_size = nn.modules.utils._pair(self.roi_layers[0].out_size)
else:
     out_size = self.roi_layers[0].output_size
```

to
```
out_size = nn.modules.utils._pair(self.roi_layers[0].out_size)
```
https://github.com/open-mmlab/mmrotate/blob/c5bf348562fd84cc17906c2cf370d1a49fcd3035/mmrotate/models/roi_heads/roi_extractors/rotate_single_level_roi_extractor.py#L102

**Comment by nijkah** [comment]
Hi @makabakasu. It is solved by this PR(https://github.com/open-mmlab/mmrotate/pull/213).
It will be nice if you close this discussion as completed. 😃
